GliaWidgets/CallVisualizer/VisitorCode/VisitorCodeView.swift

```diff
@@ -0,0 +1,172 @@
+import UIKit
+
+extension CallVisualizer {
+    internal final class VisitorCodeView: BaseView {
+        struct Props: Equatable {
+            enum ViewType: Equatable {
+                case alert(closeButtonTap: Cmd)
+                case embedded
+            }
+
+            let viewType: ViewType
+            let style: AlertStyle
+            let visitorCode: String
+            let isPoweredByShown: Bool
+
+            var isShadowNeeded: Bool {
+                switch viewType {
+                case .alert:
+                    return true
+                case .embedded:
+                    return false
+                }
+            }
+
+            var closeButtonTap: Cmd? {
+                switch viewType {
+                case let .alert(closeButtonTap):
+                    return closeButtonTap
+                case .embedded:
+                    return nil
+                }
+            }
+
+            init(
+                viewType: ViewType = .alert(closeButtonTap: .nop),
+                style: AlertStyle = .initial,
+                visitorCode: String = "11111",
+                isPoweredByShown: Bool = true
+
```

Comment:
```suggestion
```

Reply:
fixed [676c74c](https://github.com/salemove/ios-sdk-widgets/pull/448/commits/676c74ceb781899d4ca4fa8a8ab684c8a1837308)

Reply:
perhaps GitHub shows me diff incorrectly, but is empty line still there or not?